cli.js

```diff
@@ -32,8 +32,22 @@ const printAllStyles = () => {
 		'bgCyan',
 		'bgWhite',
 	];
+	allStyles.sliceByValue = function (startValue, endValue) {
```

Comment:
It's better to accept an array as argument than to add a property to the array.

Reply:
Just updated and the code no longer uses `sliceByValue` at all.